### PR TITLE
{toolchain}[dummy/dummy] intelcuda v2017a

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by shahzebsiddiqui** [comment]
@boegel

File: easybuild/easyconfigs/i/impi/impi-2017.1.132-iccifortcuda-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
```
Comment (verdurin) [inline easybuild/easyconfigs/i/impi/impi-2017.1.132-iccifortcuda-2017a.eb]: @shahzebsiddiqui please fix the outdated URL

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 10 (6 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aeb210645f307475a23c1c7437e3a730 for a full test report.

**Comment by shahzebsiddiqui** [comment]
Hey @easybuilders/easybuild-easyconfigs-maintainers last commit address the issue, link has been changed and the checks succeed in Travis. Can we merge this

**Comment by boegel** [comment]
@shahzebsiddiqui Sorry for not following up on this... Is this still relevant to merge?

**Comment by shahzebsiddiqui** [comment]
@biegel no worries you can close it. Its 2 years ago

**Comment by migueldiascosta** [comment]
@shahzebsiddiqui, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**